gst/tensor_filter/tensor_filter.c

```diff
@@ -270,10 +274,18 @@ gst_tensor_filter_class_init (GstTensorFilterClass * klass)
       g_param_spec_string ("input", "Input dimension",
           "Input tensor dimension from inner array, upto 4 dimensions ?", "",
           G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
+  g_object_class_install_property (gobject_class, PROP_INPUTNAME,
+      g_param_spec_string ("inputname", "Name of Input Tensor",
+          "The Name of Input Tensor", "",
+          G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
```

Comment:
Add comments:
```
/* INPUTNAME is required by tensorflow to designate the order of tensors */
```
For  INPUT/OUTPUT_NAME

And add usage example (pipeline using tensorflow along with multitensor input/output and names)
on the top of the code.

Reply:
it's added